README.md

```diff
@@ -25,6 +25,7 @@ Pre-built packages are available for a few Linux distros.
 * Debian 7 (Wheezy) or higher: packages in this [repository](docs/debian.md)
 * Ubuntu/Kubuntu 16.04+: use the `solaar-gnome3` and/or `solaar` package from [universe repository][universe repository]
 * Ubuntu/Kubuntu 12.04 to 14.04: use `solaar` package from [ppa:daniel.pavel/solaar][ppa]
+* Ubuntu/Kubuntu 12.04 to 19.04 git build packages: use `solaar-gnome3` and/or `solaar`  package from [ppa:solaar-unifying/ppa][ppa1]
```

Comment:
19.04 -> 20.04

Reply:
Ah, yes. Updating this should also be good.